system_tasks.yml

```diff
@@ -1,8 +1,12 @@
 ---
 - name: Update apt database
```

Comment:
I think this task can be folded into the `Install packages` task below by adding the `update_cache` option to it, right?